README.md

```diff
@@ -76,9 +76,9 @@
    # Get it via `gcloud billing accounts list` (pick the GmbH account)
    billing_account = "<our-billing-account-id>"
 
-   # Get it via `gcloud secrets versions access latest --secret relayer-private-key-staging`
+   # Get it via `gcloud secrets versions access latest --secret relayer-mnemonic-staging`
```

Comment:
should we clarify that this mnemonic is the same across staging and production

Reply:
updated